ipaserver/install/dsinstance.py

```diff
@@ -582,14 +582,15 @@ def __update_dse_ldif(self):
             'dse.ldif'
         )
 
-        with tempfile.NamedTemporaryFile(delete=False) as new_dse_ldif:
+        with tempfile.NamedTemporaryFile(
+                mode='w', delete=False) as new_dse_ldif:
```

Comment:
The default `mode` is `w+b`, so I guess if you just want to change from binary to text mode, `mode` should be `w+t`.